### PR TITLE
estimator: switch to Haswell-v4 architecture & podman work

### PR DESCRIPTION
**Pull request by nagisa**
The previous CPU of choice does not support AVX, so the estimation does not complete with that CPU of choice. However QEMU did not support these instructions in its TCG in versions used before. I had to grab a more recent version of it and build it in our reproducible environment. Now I don't appreciate docker needing a root service so I rewrote some of the text around the setup to use podman user-space container runtime instead. Now the scripts run without any notable permission or service requiremenets (besides the requirement to have podman tool available instead of docker.)

References to docker still remain here and there – I will resolve them once I'm done with the primary task at hand.

The estimations are not quite what they seem to have been recorded last time continuous estimation ran successfully (protocol version was 50!?) but this difference appears to be reasonably reproducible between the two CPU choices (which makes sense -- we're counting instructions executed) for what test cases work with the previous option.

Unfortunately it is difficult for me to provide specific numbers as the full estimator run takes some 12 hours-or-so to run under qemu…

### CODE REVIEW

**Comment by nagisa** [comment]
This is still waiting for a review @saketh-are

**Comment by saketh-are** [comment]
Sorry I missed this, taking a look now.

**Review by saketh-are — APPROVED** [review]
Approving to unblock. As discussed this doesn't seem to be portable to MacOS; perhaps it is worth noting in the readme.

**Pull request merged**
